src/annotations/components/EditAnnotationForm.tsx

```diff
@@ -34,96 +34,84 @@ import {
 } from 'src/shared/copy/notifications'
 import {notify} from 'src/shared/actions/notifications'
 
-interface EditAnnotationProps {
+interface Props {
   handleSubmit: (editedAnnotation: EditAnnotation) => void
   annotation: Annotation
   handleClose: () => void
 }
 
-export const EditAnnotationForm: FC<EditAnnotationProps> = ({
-  handleClose,
-  handleSubmit,
-  annotation,
-}) => {
+export const EditAnnotationForm: FC<Props> = (props: Props) => {
```

Comment:
not destructuring props is consistent with the rest of annotations. keeping props variables scoped to props allows easier differentiation between `props.annotation` and `updateAnnotation(annotation)`